mezzanine/core/templates/admin/base_site.html

```diff
@@ -45,6 +89,27 @@
     {% if hide_delete_link or singleton %}.submit-row .deletelink {display:none !important;}{% endif %}
     {% if hide_slug_field %}.slug {display:none !important;}{% endif %}
     {% if singleton %}.change-view-save-another {display:none !important;}{% endif %}
+
+     /* override for base product style checkboxes i.e period, style etc, hopefully this won't mess stuff up elsewhere */
+    ul#id_style li label, ul#id_period li label{ width: 100%; }
+    .form-row input[type="checkbox"] { margin: -2px 0 0;}
+    .form-row ul label { width: 100%; }
+    select#id_related_service_areas, select#id_services { width: 95%;}
+    span.page-model { float: left; font-size: 0.6rem; padding: 2px 8px 0px; text-transform: uppercase;}
+    #tree { min-width: 600px; width: 80% !important; }
+
+    .filebrowser td.fb_icon img {max-width: 150px; width: 100%; }
+    td.field-admin_thumb {max-width: 150px;} /* #STEV001 */
+    td.field-admin_thumb img {width: 100%;}
+    .selector {width: 100%;}
+    .selector-available, .selector-chosen { max-width: 500px; width: 48%;}
+    .selector-available select[multiple=multiple], .selector-chosen select[multiple=multiple]{ width: 100%; }
+    #changelist table { width: 100%; }
+    #changelist table img { width: 100%; }
+    textarea[readonly] { background: #ddd; color: #eee; }
+	table .field-available img { max-width: 20px; }
+	select[multiple="multiple"] { width: 900px; }
+
```

Comment:
Are these changes only applicable to cartridge? Can we move them there?

Reply:
L109: As part of some of our changes at Fermor we added a system that will make the meta description read only if it's automatically generated, these styles will highlight that.

L110: A project specific override for a boolean field "check" to be fixed to a maximum width, can be removed.

L111: A style change fix that we have found helpful where selects don't take up enough space, I'll remove it.